cms/celery.py

```diff
@@ -21,3 +19,18 @@
 # pickle the object when using Windows.
 APP.config_from_object('django.conf:settings')
 APP.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
+
+
+class Router(AlternateEnvironmentRouter):
+    """
+    An implmentation of AlternateEnvironmentRouter, for routing tasks to non-cms queues.
+    """
+
+    @property
+    def alternate_env_tasks(self):
```

Comment:
I went with a property here so we could make the base class have an `abstractproperty`, forcing implementing classes to override. Is having a class-specific constant better in this case?